deployment/terraform-django/app.tf

```
@@ -0,0 +1,219 @@
+#
+# Security group resources
+#
+resource "aws_security_group" "alb" {
+  name   = "sg${local.short}AppLoadBalancer"
+  vpc_id = module.vpc.id
+
+  tags = {
+    Name        = "sg${local.short}AppLoadBalancer"
+    Project     = var.project
+    Environment = var.environment
+  }
+}
+
+resource "aws_security_group" "app" {
+  name   = "sg${local.short}AppEcsService"
+  vpc_id = module.vpc.id
+
+  tags = {
+    Name        = "sg${local.short}AppEcsService",
+    Project     = var.project
+    Environment = var.environment
+  }
+}
+
+#
+# ALB Resources
+#
+resource "aws_lb" "app" {
+  name            = "alb${local.short}App"
+  security_groups = [aws_security_group.alb.id]
+  subnets         = module.vpc.public_subnet_ids
+
+  tags = {
+    Name        = "alb${local.short}App"
+    Project     = var.project
+    Environment = var.environment
+  }
+}
+
+resource "aws_lb_target_group" "app" {
+  name = "tg${local.short}App"
+
+  health_check {
+    healthy_threshold   = "3"
+    interval            = "30"
+    matcher             = "400"
```

I presume this is simply because we don't have a `/health-check/` endpoint at all yet, so the application is returning 404?

Can you create an issue for adding healthchecks (it looks like on other Django projects we typically use https://github.com/mwarkentin/django-watchman) and add a `TODO` here that references it?

---

Done, https://github.com/azavea/echo-locator/issues/441

---

Please add the TODO pointing to the Github issue as well
This is a team process we try to follow where every TODO refs a Github issue, so that
1) the TODOs are tracked by issues for planning purposes so we don't forget about them
2) you can easily search for all the TODOs that reference an issue when you're working on it

```suggestion
    # TODO GH #411: Set this to 200 after we add a health check
    matcher             = "400"
```

---

oh, did a different format, will redo with that^